packages/modular-scripts/src/lint.ts

```diff
@@ -71,6 +72,11 @@ async function lint(
         MODULAR_LINT_FIX: String(fix),
       },
     });
+
+    if (staged && fix) {
+      targetedFiles = targetedFiles.map((p) => p.replace('<rootDir>/', ''));
```

Comment:
`<rootDir>` is added for eslint purposes but that is not required for git re-staging purposes.